tmpfiles.d/legacy.conf.in

```diff
@@ -21,11 +21,4 @@ L$ /var/log/README - - - - ../..{{DOC_DIR}}/README.logs
 # /run/lock/subsys is used for serializing SysV service execution, and
 # hence without use on SysV-less systems.
 d /run/lock/subsys 0755 root root -
-
-# /forcefsck, /fastboot and /forcequotacheck are deprecated in favor of the
-# kernel command line options 'fsck.mode=force', 'fsck.mode=skip' and
-# 'quotacheck.mode=force'
-r! /forcefsck
-r! /fastboot
-r! /forcequotacheck
```

Comment:
Should these lines be kept for one extra release, to ensure any leftovers are cleaned up? Given these lines are for removals?